File: Clumsy_UFO/desktop/src/com/nopalsoft/clumsy/desktop/DesktopLauncher.java
package com.nopalsoft.clumsy.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
import com.nopalsoft.clumsy.MainClumsy;

public class DesktopLauncher {
    public static void main(String[] args) {
        LwjglApplicationConfiguration cfg = new LwjglApplicationConfiguration();
        cfg.title = "Clumsy Ufo";
        cfg.width = 480;
        cfg.height = 800;

        new LwjglApplication(new MainClumsy(), cfg);
    }
}

File: Clumsy_UFO/desktop/src/com/nopalsoft/clumsy/desktop/DesktopLauncher.kt
package com.nopalsoft.clumsy.desktop

import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration
import com.nopalsoft.clumsy.ClumsyUfoGame

object DesktopLauncher {
    @JvmStatic
    fun main(args: Array<String>) {
        val config = LwjglApplicationConfiguration()
        config.title = "Clumsy Ufo"
        config.width = 480
        config.height = 800

        LwjglApplication(ClumsyUfoGame(), config)
    }
}
